---
title: Bengtsson-Palme Lab
name: Bengtsson-Palme
picture: img/lab_bp.jpg
card: We do research in microbiology and microbial ecology, primarily focusing
  on antibiotic resistance, pathogenesis and interactions in bacterial
  communities using data-driven analysis and experiments
template: labs/single
profileleftbox: |-
  Assistant Professor  
  Johan Bengtsson-Palme

  Phone:  –   
  E-mail: johan.bengtsson.palme [at] chalmers.se  
  Office: Room 3054B (Fysik Origo, Kemigården 1)
---
Our research is centered on the ecology and evolution of opportunistic pathogens, the genes involved in virulence, competition, transfer of genetic material and antibiotic resistance, and the factors that support interactions between bacteria in microbiomes. The aim of our research is to combine data-driven bioinformatic approaches with high-throughput molecular methods and the robust theories developed for plant and animal ecology to understand pathogens, microbial communities and their interactions, in order to improve human and environmental health.

We mainly do work in three areas associated with this overarching goal: *i)* **the mechanisms behind pathogenicity**, *ii)* **the effects of disrupted interactions in microbiomes**, and *iii)* **the development and spread of antibiotic resistance**. In addition, the lab develops tools for large-scale bioinformatic analysis of high-throughput data, including the widely used ITSx and Metaxa software, to support cutting-edge research in these three areas. We also maintain a strong interest in maintaining and improving accuracy and reliability of sequence databases. You can read more on our specific projects below, or at our lab website: <https://microbiology.se>
<br/>
<br/>
##### Bacterial pathogenicity

Many of the genes involved in pathogenicity and virulence seem to have evolved from genes with other, often somewhat related, functions. Building on this, our research in this area aims to understand which genes that render bacteria pathogenic based on the hypothesis that genes involved in related processes can be repurposed as virulence factors in other contexts.
<br/>
<br/>
##### Community interactions

It is becoming increasingly clear that microbes have fundamental impacts on human and environmental health. For example, perturbations to the human microbiome have been identified to be associated with, contributing towards, or perhaps even solely causative of many human diseases, including colon cancer, depression, IBS and IBD.

We want to identify the factors that provide functional stability in microbial communities, i.e., how a consortium of microorganisms can coexist and maintain a stable relationship with each other and their host, and how that stability contributes to avoiding intrusion or uncontrolled growth by detrimental microbes such as pathogens. While the specific mechanisms that cause dysbiosis are unknown, microbial biodiversity seems to play an important role in maintaining a healthy microbiota. At the same time, very little is known about which genes that contribute to whether a community can withstand environmental disturbances or invasion events. Species interactions are a fundamental part of plant and animal ecology, which means that there exists a large theoretical framework for interactions between species, individuals and their environment, and our research aims to apply this framework to the human microbiome to better understand what influences its stability.
<br/>
<br/>
##### Antibiotic resistance

One major goal of our research on antibiotic resistance is to provide quantitative data to populate risk models for resistance emergence and propagation. Specifically, we work on determining concentrations of antibiotics that can drive selection for resistant bacteria and transfer of resistance genes. We also work on establishing the typical background levels of resistant bacteria and resistance genes in different environments, particularly those where humans interact with resistant bacteria from the environment. The lab coordinates EMBARK (<http://antimicrobialresistance.eu>), which will generate such background data from relatively pristine and human-impacted environments, as well as from the human microbiome. This will provide a baseline for future monitoring efforts for antibiotic resistance. The project also involves standardizing protocols and making different monitoring practices comparative, which will enable a much broader use of the monitoring data that is already being generated.
<br/>
<br/>
##### Methods development

The lab has been involved in several software development efforts aiming to better extract and visualize information from big datasets, such as the Metaxa, Megraft, TriMetAss, PETKit, FARAO, Mumame and CAFE software packages. Many of these deal with taxonomic analysis of microbial communities, either from shotgun metagenomic data (Metaxa and Metaxa2), or for barcode regions (Metaxa, Metaxa2 and ITSx). We are also involved in the maintenance of the UNITE database for species identification of fungi.
<br/>
<br/>
<br/>
#### Lab members

Johan Bengtsson-Palme, assistant professor  
Anna Abramova, postdoc  
Emil Burman, PhD student  
Mirjam Dannborg, PhD student  
Marcus Wenne, bioinformatician  
Lisa Teichmann, external associated PhD student  