---
name: Engqvist Lab
title: engqvist
picture: /img/lab_engqvist.jpg
card: >-
  Our research focuses on bioinformatics and experimental high-throughput
  biochemical characterization of metabolic enzymes.
template: labs/single
profileleftbox: |-
  Assistant Professor

  Martin Engqvist

  Department of Biology and Biological Engineering,

  Chalmers University of Technology,

  Kemivägen 10, SE-412 96 Göteborg, Sweden

  **Phone**: +46(0)31 772 8171

  **Fax**: +46(0)31 772 3801

  **E-mail**: martin.engqvist \[at] chalmers.se

  **Office**: Chemistry building, Room 2024
---
CV | Publications | [Software](#software)

## Publications

Our research focuses on bioinformatics and experimental high-throughput biochemical characterization of metabolic enzymes. We aim to understand how diverse protein sequences determine enzyme function.

Genome sequence data is growing at an explosive rate while experimental data relating to gene function is only growing slowly. The gap between what we know and what we do not know is increasing. The research community must greatly increase the body of experimental data to achieve accurate functional annotation of genes in current and future sequenced genomes, thereby unlocking their full value. A new generation of high-throughput experimental platforms are needed to provide this data.

Combining ideas and expertise from bioinformatics, biochemistry and directed evolution we operate a platform for high-throughput biochemical characterization of enzymes. It is a novel way of applying well-established methods towards a new purpose. We make use of these methods to densely sample sequence diversity in enzyme families. The resulting data is leveraged to gain insights into the sequence-function relationship in metabolic enzymes and forms a basis for improved functional gene annotation in sequenced genomes. 

Our research focuses on bioinformatics and experimental high-throughput biochemical characterization of metabolic enzymes. We aim to understand how diverse protein sequences determine enzyme function.

Genome sequence data is growing at an explosive rate while experimental data relating to gene function is only growing slowly. The gap between what we know and what we do not know is increasing. The research community must greatly increase the body of experimental data to achieve accurate functional annotation of genes in current and future sequenced genomes, thereby unlocking their full value. A new generation of high-throughput experimental platforms are needed to provide this data.

Combining ideas and expertise from bioinformatics, biochemistry and directed evolution we operate a platform for high-throughput biochemical characterization of enzymes. It is a novel way of applying well-established methods towards a new purpose. We make use of these methods to densely sample sequence diversity in enzyme families. The resulting data is leveraged to gain insights into the sequence-function relationship in metabolic enzymes and forms a basis for improved functional gene annotation in sequenced genomes.

## Software

Our research focuses on bioinformatics and experimental high-throughput biochemical characterization of metabolic enzymes. We aim to understand how diverse protein sequences determine enzyme function.

Genome sequence data is growing at an explosive rate while experimental data relating to gene function is only growing slowly. The gap between what we know and what we do not know is increasing. The research community must greatly increase the body of experimental data to achieve accurate functional annotation of genes in current and future sequenced genomes, thereby unlocking their full value. A new generation of high-throughput experimental platforms are needed to provide this data.

Combining ideas and expertise from bioinformatics, biochemistry and directed evolution we operate a platform for high-throughput biochemical characterization of enzymes. It is a novel way of applying well-established methods towards a new purpose. We make use of these methods to densely sample sequence diversity in enzyme families. The resulting data is leveraged to gain insights into the sequence-function relationship in metabolic enzymes and forms a basis for improved functional gene annotation in sequenced genomes.
