---
title: Engqvist Lab
name: engqvist
picture: /img/lab_engqvist.jpg
card: >-
  Our research focuses on bioinformatics and experimental high-throughput
  biochemical characterization of metabolic enzymes.
template: labs/single
profileleftbox: |-
  Assistant Professor  
  **[Martin Engqvist](/people/martin-engqvist)**

  **Phone**: +46(0)31 772 8171\
  **E-mail**: martin.engqvist \[at] chalmers.se\
  **Office**: Chemistry building, Room 2024
---
[CV](/people/martin-engqvist/) &nbsp;&nbsp;&nbsp;&nbsp; | &nbsp;&nbsp;&nbsp;&nbsp; [Publications](#publications) &nbsp;&nbsp;&nbsp;&nbsp; | &nbsp;&nbsp;&nbsp;&nbsp;[Software](#software)

<br/>

Our research focuses on bioinformatics and experimental high-throughput biochemical characterization of metabolic enzymes. We aim to understand how diverse protein sequences determine enzyme function.

Genome sequence data is growing at an explosive rate while experimental data relating to gene function is only growing slowly. The gap between what we know and what we do not know is increasing. The research community must greatly increase the body of experimental data to achieve accurate functional annotation of genes in current and future sequenced genomes, thereby unlocking their full value. A new generation of high-throughput experimental platforms are needed to provide this data.

Combining ideas and expertise from bioinformatics, biochemistry and directed evolution we operate a platform for high-throughput biochemical characterization of enzymes. It is a novel way of applying well-established methods towards a new purpose. We make use of these methods to densely sample sequence diversity in enzyme families. The resulting data is leveraged to gain insights into the sequence-function relationship in metabolic enzymes and forms a basis for improved functional gene annotation in sequenced genomes.

### Publications

As of January 2019  (Google Scholar)\
citations: 881\
h-index:  14  

<br>

#### Preprints

<em>ϕ</em> Indicates shared first-authorship as noted in publications. 

##### 2018

**"Correlating enzyme annotations with a  large set of microbial growth temperatures reveals metabolic adaptations to  growth at diverse temperatures".**\
**Engqvist MKM**\
bioRxiv (2018)\
**[doi: 10.1101/271569](https://dx.doi.org/10.1101/271569)**

<br>

#### Peer-reviewed original articles

<sup>ϕ</sup> Indicates shared first-authorship as noted in  publications  

##### 2018

**“DNA polymerase η contributes to genome-wide lagging strand synthesis”**\
Kreisel K, **Engqvist MKM,** Kalm J, Thompson LJ, Boström M, Navarrete C, McDonald JP, Larsson E, Woodgate R, Clausen AR  \
Nucleic Acids Research (2018).\
**[doi: 10.1093/nar/gky1291](https://doi.org/10.1093/nar/gky1291)**

**“Correlating enzyme annotations with a large set of microbial growth temperatures reveals metabolic adaptations to growth at diverse temperatures”**\
**MKM Engqvist**\
BMC Microbiology 18 (177) (2018).\
**[doi: 10.1186/s12866-018-1320-7](https://doi.org/10.1186/s12866-018-1320-7)**   

##### 2017

**"Ribonucleotides incorporated by the  yeast mitochondrial DNA polymerase are not repaired"**\
Wanrooij PH, **Engqvist MKM**, Forslund  JME, Navarrete C, Nilsson AK, Sedman J, Wanrooij S, Clausen AR, Chabes A.\
Proceedings of the National Academy of Sciences, Nov 7 (2017)\
**[doi: 10.1073/pnas.1713085114](https://dx.doi.org/10.1073/pnas.1713085114)**

**"Simultaneous Mapping and Quantitation  of Ribonucleotides in Human Mitochondrial DNA"**\
Kreisel, K, **Engqvist, MKM**, Clausen, AR.\
Journal of Visualized  Experiments, e56551 (2017).\
**[doi: 10.3791/56551](https://dx.doi.org/10.3791/56551)**  

**"Nucleotide pools dictate the identity  and frequency of ribonucleotide incorporation in mitochondrial DNA"**\
Berglund AK<sup>ϕ</sup>,  Navarrete C<sup>ϕ</sup>, **Engqvist MKM**, Hoberg E, Szilagyi Z, Taylor RW,  Gustafsson CM, Falkenberg M and Clausen AR.\
PLOS Genetics, 13(2): e1006628 (2017).\
**[doi:   10.1371/journal.pgen.1006628](https://dx.doi.org/10.1371/journal.pgen.1006628)**    

##### 2016

**"Highlighting the need for systems-level  experimental characterization of plant metabolic enzymes"**\
**Engqvist MKM**\
Frontiers in Plant Science, 7: 1127 (2016).\
**[doi: 10.3389/fpls.2016.01127](https://dx.doi.org/10.3389/fpls.2016.01127)**   

**"The influence of alternative pathways of  respiration that utilize branched-chain amino acids following water shortage in  Arabidopsis"**\
Pires MV, Júnior AA, Medeiros DB, Daloso DM, Pham PA, Barros KA, **Engqvist MKM**, Florian A, Krahnert I, Maurino VG,  Araújo WL, Fernie AR.\
Plant, Cell &amp; Environment, 39 (6): 1304-19 (2016).\
**[doi: 10.1111/pce.12682](https://dx.doi.org/10.1111/pce.12682)**  

##### 2015

**"ANT: software for generating and  evaluating degenerate codons for natural and expanded genetic codes"**\
**Engqvist MKM**, Nielsen J.\
ACS Synthetic Biology, 4 (8): 935–938 (2015).\
**[doi: 10.1021/acssynbio.5b00018](https://dx.doi.org/10.1021/acssynbio.5b00018)**   

**"GLYCOLATE OXIDASE3, a Glycolate Oxidase  Homolog of Yeast l-Lactate Cytochrome c Oxidoreductase, Supports l-Lactate  Oxidation in Roots of Arabidopsis"**\
**Engqvist MKM**<sup>ϕ</sup>, Schmitz J<sup>ϕ</sup>, Gertzmann A<sup>ϕ</sup>, Florian A, Jaspert N, Arif M, Balazadeh S,  Mueller-Roeber B, Fernie AR, Maurino VG.\
Plant Physiology, 169 (2): 1042-1061 (2015).\
**[doi: 10.1104/pp.15.01003](http://dx.doi.org/10.1104/pp.15.01003)**

**"Plants Possess a Cyclic Mitochondrial  Metabolic Pathway similar to the Mammalian Metabolic Repair Mechanism Involving  Malate Dehydrogenase and l-2-Hydroxyglutarate Dehydrogenase"**\
Hüdig M, Maier A, Scherrers I, Seidel L, Jansen EEW, Mettler-Altmann T, **Engqvist MKM**, Maurino VG.\
Plant and Cell Physiology, 56 (9): 1820-1830 (2015).\
**[doi: 10.1093/pcp/pcv108](https://dx.doi.org/10.1093/pcp/pcv108)**  

**"Adaptive mutations in sugar metabolism  restore growth on glucose in a pyruvate decarboxylase negative yeast  strain"**\
Zhang Y, Liu G, **Engqvist MKM**, Krivoruchko A,  Hallström BM, Chen Y, Siewers V, Nielsen J.\
Microbial cell factories, 14 (1): 116 (2015).\
**[doi: 10.1186/s12934-015-0305-6](https://dx.doi.org/10.1186/s12934-015-0305-6)**  

**"Directed Evolution of a Far-Red  Fluorescent Rhodopsin"**\
McIsaac RS<sup>ϕ</sup>,** Engqvist MKM**<sup>ϕ</sup>,  Wannier T, Rosenthal AZ, Herwig L, Flytzanis NC, Imasheva ES, Lanyi JK,  Balashov SP, Gradinaru V and Arnold FH.\
Proceedings of the National Academy of Sciences, 111 (36): 13034-13039 (2015).\
**[doi: 10.1073/pnas.1413987111](https://dx.doi.org/10.1073/pnas.1413987111)**

##### 2014

**"Archaerhodopsin  variants with enhanced voltage-sensitive fluorescence in mammalian and  Caenorhabditis elegans neurons"**\
Flytzanis NC<sup>ϕ</sup>, Bedbrook CN<sup>ϕ</sup>, Chiu H,** Engqvist MKM**, Xiao C, Chan KY,  Sternberg PW, Arnold FH, Gradinaru V.\
Nature Communications, 5(4894) (2014).\
**[doi: 10.1038/ncomms5894](https://dx.doi.org/10.1038/ncomms5894)**

**"Directed Evolution of Gloeobacter  violaceus Rhodopsin Spectral Properties"**\
**Engqvist MKM**, McIsaac RS, Dollinger P,  Flytzanis NC, Abrams M, Schor S and Arnold FH.\
Journal of Molecular Biology, 427 (1): 205-220 (2014).\
**[doi: 10.1016/j.jmb.2014.06.015](https://dx.doi.org/10.1016/j.jmb.2014.06.015)**  

##### 2013

**"D-2-hydroxyglutarate metabolism is  linked to photorespiration in the shm1-1 mutant"**\
Kuhn A<sup>ϕ</sup>,** Engqvist MKM**<sup>ϕ</sup>,  Jansen EEW, Jakobs C, Weber APM and Maurino VG.\
Plant Biology, 15(4): 776-84 (2013).\
**[doi: 10.1111/plb.12020](https://dx.doi.org/10.1111/plb.12020)**  

##### 2012

**"Transgenic Introduction of a Glycolate  Oxidative Cycle into A. thaliana Chloroplasts Leads to Growth Improvement"**\
Maier A, Fahnenstich H, von Caemmerer S,** Engqvist  MKM**, Weber APM, Flügge UI and Maurino VG.\
Frontiers in Plant Science, 3(February): 1-12 (2012).\
**[doi: 10.3389/fpls.2012.00038](https://dx.doi.org/10.3389/fpls.2012.00038)**

**"D-Lactate dehydrogenase as a marker gene  allows positive selection of transgenic plants"**\
Wienstroer J<sup>ϕ</sup>, **Engqvist MKM**<sup>ϕ</sup>,  Kunz HH<sup>ϕ</sup>, Flügge  UI and Maurino VG.\
FEBS Letters, 586(1): 36-40 (2012).\
**[doi: 10.1016/j.febslet.2011.11.020](https://dx.doi.org/10.1016/j.febslet.2011.11.020)**   

##### 2011

**"Plant D-2-hydroxyglutarate dehydrogenase  participates in the catabolism of lysine especially during senescence"**\
**Engqvist  MKM**<sup>ϕ</sup>,  Kuhn A<sup>ϕ</sup>,  Wienstroer J, Weber K, Jansen EE, Jakobs C, Weber APM and Maurino VG.\
Journal of Biological Chemistry, 286(13): 11382-90 (2011).\
**[doi: 10.1074/jbc.m110.194175](https://dx.doi.org/10.1074/jbc.m110.194175)**  

##### 2009

**"Two D-2-hydroxyacid dehydrogenases in  Arabidopsis thaliana with catalytic capacities to participate in the last  reactions of the methylglyoxal and β-oxidation  pathways"**\
**Engqvist M**, Drincovich MF, Flügge UI and  Maurino VG.\
Journal of Biological Chemistry, 284(37): 25026-37 (2009).\
**[doi: 10.1074/jbc.m109.021253](https://dx.doi.org/10.1074/jbc.m109.021253)**  

##### 2008

**"HAG2/MYB76 and HAG3/MYB29 exert a specific  and coordinated control on the regulation of aliphatic glucosinolate  biosynthesis in Arabidopsis thaliana"**\
Gigolashvili T<sup>ϕ</sup>, **Engqvist M**<sup>ϕ</sup>, Yatusevich R, Müller C, Flügge UI.\
New Phytologist 177 (3), 627-642 160 (2008)\
**[doi:  10.1111/j.1469-8137.2007.02295.x](https://doi.org/10.1111/j.1469-8137.2007.02295.x)**  

##### 2007

**"Effect of poly (ethylene glycol) on enzymatic  hydrolysis and adsorption of cellulase enzymes to pretreated lignocellulose"**\
Börjesson J, **Engqvist M**, Sipos B,  Tjerneld F.\
Enzyme and Microbial Technology 41 (1),  186-195 (2007)\
**[doi:  10.1016/j.enzmictec.2007.01.003](https://doi.org/10.1016/j.enzmictec.2007.01.003)**

<br>

#### Review articles

**“Applications of protein engineering and directed evolution in plant research”**\
**Engqvist MKM,** Rabe KS\
Plant Physiology, in print (2019).

**“Biochemical control systems for small molecule damage in plants”**\
Hüdig M, Schmitz J, **Engqvist MKM,** VG Maurino\
Plant signaling & behavior 13 (5), e1477906 (2018).\
**[doi: 10.1080/15592324.2018.1477906](https://doi.org/10.1080/15592324.2018.1477906)**

**"Mitochondrial 2-hydroxyglutarate  metabolism"**\
**Engqvist MKM**, Eßer C, Maier A, Lercher MJ and  Maurino VG.\
Mitochondrion. 19: 275-281 (2014).\
**[doi: 10.1016/j.mito.2014.02.009](https://dx.doi.org/10.1016/j.mito.2014.02.009)**  

<br>

#### Book chapters

**"Metabolic engineering of  photorespiration - bypass approaches"**\
**Engqvist MKM**, Maurino VG.\
In Methods in Molecular Biology: Photorespiration (eds. Fernie AR, Weber A, Bauwe H) 1653:137-155 (2017)\
**[doi: 10.1007/978-1-4939-7225-8_10](https://doi.org/10.1007/978-1-4939-7225-8_10)**

**"2-Hydroxy Acids in Plant  Metabolism"**\
Maurino VG, **Engqvist MKM**.\
In The Arabidopsis Book, Sep 4, 13:e0182 (2015).\
**[doi:  10.1199/tab.0182](https://dx.doi.org/10.1199/tab.0182)**

<br/><br/>

### Software

**Ambiguous Nucleotide Tool (ANT)**\
Degenerate codons are used to represent DNA positions that have multiple possible nucleotide alternatives. This is useful for protein engineering and directed evolution, where primers specified with degenerate codons are used as a basis for generating libraries of protein sequences. [ANT](https://github.com/mengqvist/ANT) provides Python code for generating and evaluating degenerate codons for natural and expanded genetic codes.

**DNApy**\
Manually editing a large number of plasmid files, to annotate them with mutations discovered in the lab or other features, is tedious and error-prone. [DNApy](https://github.com/mengqvist/DNApy) provides Python code for DNA and plasmid editing. It allows for batch in silico cloning of an unlimited number of sequences from FASTA files as well as the batch introduction and annotation of mutations.

**colcol**\
Making accurate color scales for communicating data in visualizations is critical. In many presentations and publications unbalanced scales are used. A common error is to use color scales which does not have smooth and constant change in lightness, leading to difficulty in interpreting the visualized data. [colcol](https://github.com/mengqvist/colcol) provides Python code to deal with color conversions as well as with generating publication-quality color scales.

**wsvg**\
Many good plotting libraries exist for Python, but sometimes it is necessary to generate custom vector art visualizations of data. [wsvg](https://github.com/mengqvist/wsvg) is a light Python wrapper around the svg (scalable vector graphics) format, enabling scripting of svg files.

**Reproducible research**\
Computational analysis of data in a research setting is often difficult to reproduce due to poor documentation of steps taken and software used. Determining which version of what script was used to process a raw data file to generate publishable results can be challenging. As ever-increasing amounts of data is being generated in biological research it is critical that this problem is addressed and that computational analysis becomes as reproducible as laboratory experiments. The [reproducible research](https://github.com/EngqvistLab/reproducible-research-init) repository provides a folder structure and conventions aimed at achieving such reproducibility.
Note: This repository was originally created by others and made public [here](https://github.com/Reproducible-Science-Curriculum/rr-init). The Engqvist Lab version has been modified from this source to fit our own research needs.

**Reproducible reports**\
Scientific writing should not be bound to any given proprietary software or file format, especially one which is ill-suited for handling complex texts and large figures. Unfortunately many scientific publishers have built their publication pipelines around such software. An alternative in many journals is to submit LaTeX documents, by this is intimidating to most users. [Pandoc](http://pandoc.org/) offers a great alternative. By writing your documents using [markdown](http://pandoc.org/MANUAL.html) Pandoc can convert that text into a whole range of common file formats, including LaTeX, pdf, doc, and others. The [reproducible reports](https://github.com/EngqvistLab/reproducible-reports-init) repository sets up a folder structure and utilities for using Pandoc to write reports.
