---
title: Kerkhoven Lab
name: kerkhoven
picture: img/lab-kerkhoven.jpg
card: Through model-driven analysis of experimental data we aim to understand,
  predict and engineer biology, among others in baker's and oleaginous yeasts
  and antibiotic producing bacteria.
template: labs/single
profileleftbox: |-
  Docent  
  **Eduard Kerkhoven**

  Phone: -  
  E-mail: eduardk [at] chalmers.se  
  Office: Room 3054A (Fysik Origo, Kemig√•rden 1)
---
[CV](/img/cv_ejk2006.pdf) &nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp; [Publications](/img/pub_ejk2006.pdf) <br/><br/>  



Our research revolves around computational metabolic engineering, where model-driven analysis of experimental data is used to understand, predict and engineer biology.

With a particular focus on metabolism we bridge the gap between *in silico* prediction and *in vivo* validation through genetic engineering. We are working on a variety of different projects, both in computational dry-lab and experimental wet-lab.

Most of our work aims to develop microbes as cell factories for sustainable production of chemicals. We work a lot with oleaginous yeasts (such as *Yarrowia lipolytica*) as they are able to accumulate large amount of lipids, and these lipids can either be directly used as product, or we rewire the metabolic network to produce other high-value chemicals.

Computational analysis of metabolism helps us to come up with strategies for metabolic engineering. We reconstruct and curate genome-scale metabolic models (GEMs) for various organisms (yeasts, bacteria, human) using our RAVEN Toolbox. These models are combined with omics analyses (primarily RNAseq and proteomics), either directly or through the use of enzyme-constrained models.
