---
title: Kerkhoven Lab
name: kerkhoven
picture: /img/lab-kerkhoven.jpg
card: Through model-driven analysis of experimental data we aim to understand,
  predict and engineer biology, among others in baker's and oleaginous yeasts
  and antibiotic producing bacteria.
template: labs/single
profileleftbox: |-
  Docent  
  **Eduard Kerkhoven**

  Phone: -  
  E-mail: eduardk [at] chalmers.se  
  Office: Room 3054A (Fysik Origo, Kemigården 1)
  [Publications](/img/pub_ejk2403.pdf)
---

### Kerkhoven lab – who are we and what do we do?

## Metabolic systems biology
The research in Kerkhoven lab revolves around metabolic systems biology, where computational model-driven analysis of experimental data is used to understand, predict and engineer biology. With a particular focus on metabolism we bridge the gap between in silico prediction and in vivo validation through data-driven genetic engineering. We are working on a variety of different projects, from developing microbes as cell factories for sustainable production of chemicals, to investigating metabolic aspects in human disease.

## Computational metabolic engineering
Computational analysis of metabolism helps us to come up with strategies for metabolic engineering. We reconstruct and curate genome-scale metabolic models (GEMs) for various organisms (yeasts, bacteria, human) using our [RAVEN Toolbox](https://github.com/SysBioChalmers/RAVEN). Our model development is tracked on [GitHub](https://github.com/search?q=org%3ASysBioChalmers+%22-GEM%22&type=Repositories), and important models are those for [*S. cerevisiae*](https://github.com/SysBioChalmers/yeast-GEM), [*Y. lipolytica*](https://github.com/SysBioChalmers/Yarrowia_lipolytica_W29-GEM), [*S. coelicolor*](https://github.com/SysBioChalmers/Sco-GEM) and [*Homo sapiens*](https://github.com/SysBioChalmers/human-GEM). These models are combined with omics analyses (primarily RNAseq and proteomics), either directly or through the use of enzyme-constrained models using our [GECKO Toolbox](https://github.com/SysBioChalmers/GECKO). In addition to biotechnological applications, we have also been using our approaches to investigate for instance evolution of the yeast subphylum, and prediction of kcat values through deep learning.

## Oleaginous yeasts as biocatalysts
Besides computational research, we also investigate the oleaginous yeast *Y. lipolytica* as microbial cell factory, for instance to produce itaconic acid. This promising platform chemical can be used as monomer to e.g. aid bioleaching, or as a range of innovative polymers. We perform this through genetic engineering, integrative omics analysis, modeling of metabolism and fermentation optimization.

**Current members:**  
Cheewin Kittikunapong, PhD student  
Simone Zaghen, PhD student  

**Previous members:**  
*Postdocs*  
Albert Tafur Rangel (2021-2023)  
Jing Fu (2019-2023)  
Naghmeh Poorinmohammad (2019-2021)  
Ievgeniia Tiukova (2018-2019)  
Hao Wang (2016-2018)  

*PhD students*  
Le Yuan (2019-2023)  

*Visiting researchers*  
Nowrouzi Behnaz (2021-2022)  
Piotr Hapeta (2018)  
Patrice Lubuta (2017)  
Xiaojun Ji (2016-2017)  

*Master students*  
Andreas Gällblad (2022-2023)  
Alessandro Fossen (2022-2023)  
Hanna Östermark (2021-2022)  
Alexander Kornberg (2021)  
Bob Wabeke (2019-2020)  
Philip Gorter de Vries (2018-2019)  
Francisco Zorrilla (2018)  
Severo Balasbas (2018)  
John Hellgren (2016-2017)  

