---
title: Kerkhoven Lab
name: kerkhoven
picture: img/lab-kerkhoven.jpg
card: Through model-driven analysis of experimental data we aim to understand,
  predict and engineer biology, among others in baker's and oleaginous yeasts
  and antibiotic producing bacteria.
template: labs/single
profileleftbox: |-
  Docent  
  **Eduard Kerkhoven**

  Phone: -  
  E-mail: eduardk [at] chalmers.se  
  Office: Room 3054A (Fysik Origo, Kemigården 1)
---
[CV](/img/cv_ejk2109.pdf) &nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp; [Publications](/img/pub_ejk2109.pdf) <br/><br/>  

### Kerkhoven lab – who are we and what do we do?

Our research revolves around computational metabolic engineering, where model-driven analysis of experimental data is used to understand, predict and engineer biology.
With a particular focus on metabolism we bridge the gap between *in silico* prediction and *in vivo* validation through genetic engineering. We are working on a variety of different projects, both in computational dry-lab and experimental wet-lab.

Most of our work aims to develop microbes as cell factories for sustainable production of chemicals. We work a lot with oleaginous yeasts (such as *Yarrowia lipolytica* and *Rhodotorula toruloides*) as they are able to accumulate large amount of lipids, and these lipids can either be directly used as product, or we rewire the metabolic network to produce other high-value chemicals. Other microbes we have projects on include baker's yeast (*Saccharomyces cerevisiae*) and the bacteria *Pseudomonas putida* and *Streptomyces* species.

Computational analysis of metabolism helps us to come up with strategies for metabolic engineering. We reconstruct and curate genome-scale metabolic models (GEMs) for various organisms (yeasts, bacteria, human) using our [RAVEN Toolbox](https://github.com/SysBioChalmers/RAVEN). Our model development is tracked on [GitHub](https://github.com/search?q=org%3ASysBioChalmers+%22-GEM%22&type=Repositories), and important models are those for [*S. cerevisiae*](https://github.com/SysBioChalmers/yeast-GEM), [*Y. lipolytica*](https://github.com/SysBioChalmers/Yarrowia_lipolytica_W29-GEM), [*S. coelicolor*](https://github.com/SysBioChalmers/Sco-GEM) and [*Homo sapiens*](https://github.com/SysBioChalmers/human-GEM). These models are combined with omics analyses (primarily RNAseq and proteomics), either directly or through the use of enzyme-constrained models using our [GECKO Toolbox](https://github.com/SysBioChalmers/GECKO). In addition to biotechnological applications, we have also been using our approaches to investigate for instance evolution of the yeast subphylum, and prediction of kcat values through deep learning.

**Current members of Kerkhoven lab:**\
Jing Fu, postdoc\
Cheewin Kittikunapong, PhD student\
Le Yuan, PhD student\
Simone Zaghen, PhD student\
Hanna Östermark, MSc student\
