---
title: Larsson Lab
name: larsson
picture: /img/lab_larsson.jpg
card: >-
  Our lab is focused on understanding the physiology and metabolic regulation in
  the yeast Saccharomyces cerevisiae and to apply this knowledge in various
  metabolic engineering projects.
template: labs/single
profileleftbox: |-
  Professor

  Christer Larsson



  Department of Biology and Biological Engineering,

  Chalmers University of Technology,

  Kemivägen 10, SE-412 96 Göteborg, Sweden



  Phone: +46(0)31 772 3806

  Fax: +46(0)31 772 3801

  E-mail: christer.larsson \[at] chalmers.se

  Office: Room 3054B
---
Our research activities are focused on developing the yeast Saccharomyces cerevisiae into an efficient producer (cell factory) of various chemicals and fuels by using a metabolic engineering approach. We have a long standing history of research in connection to optimization of bioethanol production. In more recent years we have also included butanol has an alternative biofuel that has many advantages compared to ethanol. S. cerevisiae is not a natural producer of butanol but by identification and insertion of suitable genes followed by protein engineering and careful selection of process conditions a novel efficient butanol forming organism/process will be developed. A similar approach has been used to transform S. cerevisiae into an ethylene producing organism. The aim of this project is to develop a sustainable process using renewable substrates for production of polyethylene that nowadays rely on finite fossil resources. In order to be successful in our metabolic engineering projects we also focus on exploring the mechanisms for regulation/control of various metabolic pathways in the cell. Special attention has been directed towards the role of energy metabolism and the importance of adenine nucleotide levels but also redox metabolism is a field of interest. The field of metabolic engineering in most cases relies on changing/adding the expression of proteins/genes in the host organism. To achieve proper expression and/or localization of the targetted protein it is necessary to understand the different layers of gene regulation involved. One project related to basic research into this area concerns the role(s) of, and regulatory mechanisms acting via the 3' untranslated region, using yeast as a model organism.





**Current projects**



**Optimization of industrial ethanol production.**

We have a long history of improving various aspects of industrial ethanol production. Currently we are focusing on (i) the impact of high gravity conditions on the physiology of the yeast, (ii) behaviour of different strains under different conditions and (iii) the interplay between pretreatment of lignocellulosic material and the subsequent fermentation.

**People**: Christer Larsson (Project leader), Emma Johansson (Ph.D. student), Venkata Soudham (Ph.D. student)





**Optimization of ethylene production by metabolic engineering of the yeast _Saccharomyces cerevisiae_.**

Ethylene is one of the most used bulk chemicals in modern industry (e.g. in the plastics industry). However, since its production is based on the non-renewable supply of oil, it is desirable to find alternative production strategies. We have recently demonstrated that S. cerevisiae can be transformed into a producer of ethylene by the insertion of a bacterial ethylene forming enzyme (Pirkov et al, 2008). The main goal of this project is now to achieve higher yields of ethylene. This will be performed by e.g. metabolomic analysis coupled to predictive modelling followed by metabolic engineering of predicted target(s), by enzyme improvement and by optimization of growth parameters.

People:Christer Larsson and Joakim Norbeck (Project leaders), Nina Johansson (Ph.D. student)





**Biobutanol production by metabolic engineering of _Saccharomyces cerevisiae_.**

There is a strong interest in developing "green" alternatives to gasoline due to rising prices and climate change. Ethanol has traditionally had a strong position in this respect, due to the well established and comparatively cheap procedures for its production by yeast fermentations. But higher alcohols (such as butanol) would actually be more suitable as fuel, due to, e.g. their higher energy density and less corrosive properties.  Traditionally, butanol is produced either chemically or from Clostridial fermentations (yielding 1-butanol). However, 1-butanol is highly toxic to most microorganisms, which limits the yields. We have therefore started a project to engineer yeast for production of 2-butanol from a glucose-based media, a process which is predicted to be redox neutral. The major tasks are (1) to improve tolerance to 2-butanol, (2) to identify and subsequently express the two genes encoding protein activities required for formation of 2-butanol, and (3) to optimize growth conditions and metabolism (the latter to be achieved by metabolic engineering of pathways predicted to be relevant).

**People**: Christer Larsson and Joakim Norbeck (Project leaders), Payam Ghiaci (Ph.D. student)





**Regulation of Puf-proteins and their target 3'UTR sequences in _Saccharomyces cerevisiae_.**

Optimization of protein expression in metabolic engineering applications requires knowledge of the regulatory mechanisms acting on all levels of gene regulation. Traditionally, this has involved choosing a proper promotor. However, regulation via specific motifs in the 3’untranslated region (3’UTR) on mRNA can also strongly affect the stability, localization and translation of mRNA, and hence also correct protein expression and localization. A full understanding of 3’UTR dependent regulation requires knowledge of the proteins and signalling pathways involved. We are therefore developing methodology suited to performing genome wide screening in yeast to find factors affecting specific motifs in 3’UTR reporter gene expression, which has to our knowledge never been achieved previously in any organism. We have chosen to use a flow-cytometry based reporter system, in which two fluorescent proteins are expressed under the control of 3’UTR sequences with and without point mutations in specific motifs. In the development phase, we are focusing on finding regulatory mechanisms for the S. cerevisiae RNA-binding proteins Puf3 and Puf5, for which the binding site and many putative target transcripts are known.

**People**: Joakim Norbeck (Project leader)
