---
title: Zelezniak Lab
name: zelezniak
picture: /img/lab_zelezniak.jpg
card: >-
  We apply best practices of computational biology together with machine
  learning to understand the behavior of metabolic networks at single and
  multispecies levels.
template: labs/single
profileleftbox: |-
  Assistant Professor\
  **Aleksej Zelezniak**

  Phone: +46(0)31 772 8171\
  E-mail: alezel \[at] chalmers.se\
  Office: Room 2064A (Fysik Origo, Kemigården 1)
---
Metabolism is a primordial biological system responsible for interconversion of environmental chemicals to energy and cellular building blocks in all living species. Whenever organisms are genetically affected or facing new environment they do need to adapt their metabolism to satisfy their individual growth demands. At the same time, in natural environments, organisms never exist in isolation but rather constantly interacting with other species. Ranging from local competition for the same resources up to emergent global social interactions, metabolism shapes individual species behavior and provides a common communication platform of all living entities. At Zelezniak lab we are interested in studying how genetic, environmental factors affect operation and regulation of cellular metabolic networks. At the single species level, we want to understand how complex phenotypes emerge from the underlying molecular levels organized via central biological dogma. At the multicellular level, we want to understand what is the role of metabolism in the cell-to-cell interactions, in particular, its role in the co-existence of microbial species. Answering these questions will allow us not only to design organisms with desired metabolic properties for biotechnology purposes but also engineer synthetic microbial communities with specific health benefits.

We are combining best practices of data science with artificial intelligence and metabolic modeling to develop novel technologies for getting insights about biological mechanisms directly from molecular data. Furthermore, together with our collaborators, we are actively involved in developing mass spectrometry technology to enable fast, robust, precise and inexpensive proteome acquisitions from any biological sample. At the same time, we are actively interested in expanding the scope of our computational methods applications with the goal to gain mechanistic insights about the following biological processes: the role of metabolic interactions in shaping healthy gut microbiome, designing stable microbial communities for bioremediation and other biotechnological applications, the role of tissue cross-talk in cancer metabolism. Currently, we are actively seeking collaborations within Swedish university/hospital networks and internationally.

<br/>

##### Relevant Publications

Metabolic dependencies drive species co-occurrence in diverse microbial communities  
A Zelezniak, S Andrejev, O Ponomarova, DR Mende, P Bork, KR Patil  
Proceedings of the National Academy of Sciences 112 (20), 6449-6454, 2015  

The metabolic background is a global player in Saccharomyces gene expression epistasis  
MT Alam, A Zelezniak, M Mülleder, P Shliaha, R Schwarz, F Capuano, et al  
Nature microbiology 1, 1503020, 2016  

Contribution of network connectivity in determining the relationship between gene expression and metabolite concentration changes  
A Zelezniak, S Sheridan, KR Patil PLoS computational biology 10 (4), e1003572, 2014  

Metabolic network topology reveals transcriptional regulatory signatures of type 2 diabetes  
A Zelezniak, TH Pers, S Soares, ME Patti, KR Patil  
PLoS computational biology 6 (4), e1000729, 2010  

Functional metabolomics describes the yeast biosynthetic regulome  
M Mülleder, E Calvani, MT Alam, RK Wang, F Eckerstorfer, A Zelezniak, M Ralser  
Cell 167 (2), 553-565. E12, 2016  

<br/>
<a href="https://www.scilifelab.se/research/fellows/" target="_blank">
  <img src="/img/logo_scilifelab_az.png" alt=""height="80">
</a>
